js/libs/keycloak-js/lib/keycloak.js

```diff
@@ -298,25 +298,8 @@ function Keycloak (config) {
             }
         }
 
-        function domReady() {
```

Comment:
Since scripts loaded with `type=module` will implicitly wait for the DOM to be ready there is no need to wait anymore, so I removed this code.